styles/globals.css

```diff
@@ -26,4 +26,8 @@
       @apply container mx-auto px-4;
     }
   }
-}
\ No newline at end of file
+}
+
+.notification-blue-text {
```

Comment:
Can you please add this to the tailwind config file as a color and style it that way? This blue color just for the notification text wouldn't be considered a global item

Reply:
took care of style and as of the separating of PR will do that to future stories, on PR per story.